jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -0,0 +1,209 @@
+;; -*- mode: lisp;-*-
+
+(require "package://baxtereus/baxter-interface.l")
+(require "package://jsk_2015_05_baxter_apc/euslisp/jsk_2015_05_baxter_apc/util.l")
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l")
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l")
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+
+(ros::load-ros-manifest "jsk_arc2017_baxter")
+
+(unless (find-package "JSK_ARC2017_BAXTER")
+  (make-package "JSK_ARC2017_BAXTER"))
+
+(defclass jsk_arc2017_baxter::baxter-interface
+  :super baxter-interface
+  :slots (tfl-
+          rarm-pressure-threshold-
+          larm-pressure-threshold-))
+
+(defmethod jsk_arc2017_baxter::baxter-interface
+  (:init
+    (&rest args &key
+           ((:moveit-environment mvit-env)
+            (instance jsk_2016_01_baxter_apc::baxter-moveit-environment))
+           ((:moveit-robot mvit-rb) (instance jsk_2016_01_baxter_apc::baxter-robot :init))
+           &allow-other-keys)
+    ;; initialize controllers
+    (send-super* :init :joint-states-queue-size 2
+                 :robot jsk_2016_01_baxter_apc::baxter-robot
```

Comment:
I want to use `jsk_2016_01_baxter_apc::baxter-robot` instead of `baxter-robot` for `robot` in `baxter-interface`